ui/pages/confirmations/confirm-transaction/confirm-token-transaction-switch.js

```diff
@@ -179,7 +183,10 @@ export default function ConfirmTokenTransactionSwitch({ transaction }) {
             transaction={transaction}
             ethTransactionTotal={ethTransactionTotal}
             fiatTransactionTotal={fiatTransactionTotal}
-            hexMaximumTransactionFee={hexMaximumTransactionFee}
+            hexMaximumTransactionFee={sumHexes(
+              hexMaximumTransactionFee,
+              layer1GasFee,
```

Comment:
Does `sumHexes` handle `undefined` or `null` or do we need a fallback like `?? '0x0'` here as we do elsewhere?